src/ralph/export_to_ng/management/commands/export_model.py

```diff
@@ -125,6 +141,11 @@ class Command(BaseCommand):
             action="append",
             type="str"
         ),
+        make_option(
+            '--exclude-networks',
+            help='Excludes networks from passed range, eg.: --exclude-networks=<min-ip>,<max-ip>', # noqa
```

Comment:
min and max ips are "ok", but IMO it'd be better to pass network with mask here (ex. 10.20.30.00/24).
Btw what if I want to exclude many networks? Consider `action="append"` here